systemtest/src/test/java/io/strimzi/systemtest/Resources.java

```diff
@@ -74,6 +74,7 @@
     private static final long TIMEOUT_FOR_DEPLOYMENT_CONFIG_READINESS = Duration.ofMinutes(7).toMillis();
     private static final long TIMEOUT_FOR_RESOURCE_CREATION = Duration.ofMinutes(3).toMillis();
     public static final long TIMEOUT_FOR_RESOURCE_READINESS = Duration.ofMinutes(7).toMillis();
+    private static final String KAFKA_VERSION = "2.1.0";
```

Comment:
We have this value already in `TestUtils.changeOrgAndTag()` guess we should have this only on one place. Also I would prefer to use this only as default if env variable with versions in not set (as it is in `changeOrgAndTag`).

Reply:
I removed this value in `TestUtils.changeOrgAndTag()` in this PR ;) Look on changes again plz. As for having env variable, it can be a good idea, let me do it.